src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Definition/RazorDefinitionEndpoint.cs

```diff
@@ -109,6 +109,12 @@ public async Task<LocationOrLocationLinks> Handle(DefinitionParams request, Canc
                     Range = new Range(new Position(0, 0), new Position(0, 0)),
                 }),
             });
+
+            static bool IsAttributeDescriptor(TagHelperDescriptor d)
```

Comment:
This would actually be nice to have an extension method that we have on TagHelperDescriptor's!